### PR TITLE
Add custom messages to command rewards (closes #263)

### LINKED ISSUE #263: Custom Reward Display
Is there a way display a custom command reward? As in, stead of it auto interpreting the reward from vanilla, I can put custom text as the displayed reward. 

[CODE]  3:
    Message: Level 3 reached!
    Name: maxlevel_3
    DisplayName: "&4Newbie"
    Reward:
      Command: jobs editpoints add PLAYER 5; give PLAYER 126 5
[/CODE]

Shows up as "other" for reward

### PR DESCRIPTION
**Pull request by supertassu**
Here we go. This PR closes #263.
Pretty simple, examples included.
~~Forgot to add magic words to the commit message, you'll have to close the issue manually.~~ It registers from the commit message nowadays! Cool :P 

_(on a hurry making this message, sorry! I'll try to make better sometime soon)_

### CODE REVIEW

File: src/main/java/com/hm/achievement/utils/RewardParser.java [hunk 1]
```diff
@@ -241,6 +255,25 @@ public ItemStack getItemReward(String configAchievement) {
...
+	 * @author tassu
+	 * @param configAchievement
+	 * @return the custom command message (null if not present)
+	 */
+	public String getCustomCommandMessage(String configAchievement) {
+		if (!plugin.getPluginConfig().isConfigurationSection(configAchievement + ".Reward.Command")) {
+			return null;
+		}
+
+		if (!plugin.getPluginConfig().contains(configAchievement + ".Reward.Command.Display")) {
```
Comment (PyvesB) [inline src/main/java/com/hm/achievement/utils/RewardParser.java]: I don't think this `if` block is actually needed, as the `getString` method returns `null` if the value is not found.

File: src/main/resources/config.yml [hunk 2]
```diff
@@ -265,6 +265,10 @@ Places:
       Message: "&4One thousand placed stones!"
       Name: place_1000_stone
       DisplayName: Stone God
+      Reward:
+        Command:
+          Command: "say §7ALERT: §4PLAYER§7 has become the §4STONE GOD§7!"
```
Comment (PyvesB) [inline src/main/resources/config.yml]: In my opinion having two nested `Command` sections is confusing. Could we rename the new inner-most one to something like `Execute`?

File: src/main/resources/lang.yml [hunk 3]
```diff
@@ -109,6 +110,7 @@ list-reward: "Reward(s):"
 list-reward-money: "receive AMOUNT"
 list-reward-item: "receive AMOUNT ITEM"
 list-reward-command: "other"
+list-reward-command-custom-message: "MESSAGE"
```
Comment (PyvesB) [inline src/main/resources/lang.yml]: Should we actually be giving users this new option? I think that having a custom `Display` option already gives them all the freedom they need, how would this additional configuration option prove useful? Having more options tends to be confusing and makes things more complex, I tend to only add them when they're really needed.

File: src/main/resources/config.yml [hunk 4]
```diff
@@ -265,6 +265,10 @@ Places:
...
+          Display: "Become the Stone God!"
```
Comment (PyvesB) [inline src/main/resources/config.yml]: Quotes not needed. ^^

File: src/main/resources/config.yml (hunk 2 above)
Comment (supertassu) [reply src/main/resources/config.yml]: Yep, sounds fine.

File: src/main/resources/config.yml (hunk 4 above)
Comment (supertassu) [reply src/main/resources/config.yml]: Oh.

File: src/main/resources/lang.yml (hunk 3 above)
Comment (supertassu) [reply src/main/resources/lang.yml]: Yeah. There isn't a a lot of usage cases. Sounds fine.

**Comment by supertassu** [comment]
Hey,

Thanks for the feedback. Sadly, school is a thing, so I can't fix it right now.

Tassu

**Comment by PyvesB** [comment]
No problem, there is no rush. 😉

**Comment by supertassu** [comment]
@PyvesB I've fixed these issues.

**Comment by PyvesB** [comment]
Looks good, thanks. Merged! 👍

**Pull request merged**